canopy/canopyjs/package.json

```diff
@@ -5,12 +5,17 @@
   "author": "Cargill Incorporated",
   "license": "Apache-2.0",
   "main": "dist/index.js",
-  "dependencies": {},
+  "dependencies": {
+    "@types/secp256k1": "^3.5.0",
+    "protobufjs": "^6.8.8",
+    "secp256k1": "^3.7.1"
+  },
   "scripts": {
     "test": "jest",
     "build": "tsc -p ./tsconfig.json --strict --outDir dist",
     "format": "prettier --write \"**/*.+(ts|*rc|json|js)\"",
-    "lint": "eslint src/*"
+    "lint": "eslint src/**/**/**/*",
+    "generate-proto-files": "node scripts/compile_protobuf.js ./protos"
```

Comment:
Should probably add this as a `postinstall` script

Reply:
added